tests/unittests/graphOptzTest.cpp

```diff
@@ -1061,6 +1061,42 @@ TEST_F(GraphOptz, sinkTransposeBelowConcatWithPredicate) {
   EXPECT_EQ(F_->getNodes().size(), 3);
 }
 
+TEST_F(GraphOptz, sinkTransposeBelowPad) {
+  // Before optims
+  const size_t origDims[] = {1, 5, 10, 15};
+  const size_t outTransposeDims[] = {1, 10, 15, 5};
+  int pads[] = {0, 2, 3, 1, 4, 6, 7, 5};
+  const size_t outPadDims[] = {5, 18, 25, 11};
+
+  // After optims
```

Comment:
// The shape of the graph after the optimizations.

Reply:
Done.